cleverpush/src/main/java/com/cleverpush/AddSubscriptionTags.java

```diff
@@ -17,17 +19,20 @@
     private String[] tagIds;
     private String subscriptionId;
     private String channelId;
+    private SharedPreferences sharedPreferences;
+    Set<String> tags;
```

Comment:
Check if global variable needed.

Reply:
yes we need it because we are accessing it from the test case to check the values and size of tags